presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/docker-command.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 
 echo "[$(date)] $0: configuring hadoop services"
 
-/docker/presto-product-tests/conf/docker/files/apply-site-xml-override.sh /etc/hive/conf/hive-site.xml /docker/presto-product-tests/conf/docker/singlenode-hdp3/hadoop-master/hive-site-overrides.xml
+/docker/presto-product-tests/conf/docker/files/apply-site-xml-override.sh /etc/hive/conf/hive-site.xml /docker/presto-product-tests/conf/environment/singlenode-hdp3/hive-site-overrides.xml
```

Comment:
as the script is installed as an entrypoint, it now should end with `exec "$@"` instead of `exec supervisord -c /etc/supervisord.conf`